xc/domain.ml

```diff
@@ -866,7 +866,10 @@ type suspend_flag = Live | Debug
       "-store_port"; string_of_int store_port;
       "-console_port"; string_of_int console_port;
       "-fork"; "true";
-    ] @ extras @ vgpu_cmdline
+    ]
+    @ (if Device_common.is_upstream_qemu domid then ["-dm"; "qemu"] else [])
```

Comment:
Is `Device_common.is_upstream_qemu` actually right? This reads `"/libxl/%d/dm-version" domid` from xenstore, but at this point in resume, QEMU isn't running yet.

Also, the only other place where is function is used is in `Device.Profile`, and there is a comment `(* XXX remove again *)`...